### PR TITLE
fix for showSignatureVariances (showSignatureDistribution) with 1 mode

### PR DESCRIPTION
**Pull request by jamesmkrieger**
Otherwise the error is

```
showSignatureVariances(anm_ens_20[:,0], fract=False)                                                           
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-25-c113ce977cda> in <module>
----> 1 showSignatureVariances(anm_ens_20[:,0], fract=False)

~/Documents/code/ProDy/prody/dynamics/signature.py in showSignatureVariances(mode_ensemble, **kwargs)
   1562     labels = ['mode %d'%(i+1) for i in indices][::-1]
   1563 
-> 1564     show = showSignatureDistribution(sig, label=labels, **kwargs)
   1565 
   1566     xlabel('Variance')

~/Documents/code/ProDy/prody/dynamics/signature.py in showSignatureDistribution(signature, **kwargs)
   1507         figure(fig_num)
   1508 
-> 1509     W = signature.getArray()[:, ::-1] # reversed to accommodate with matplotlib.pyplot.hist
   1510     weights = np.ones_like(W)/float(len(W))
   1511 

IndexError: too many indices for array
```

### CODE REVIEW

**Comment by jamesmkrieger** [comment]
We still have another error downstream of that though so I don't think that's the right fix actually.

```
In [4]: plt.figure(); showSignatureVariances(anm_ens_20[:,0])                                                           
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-4-9279eeed7a8b> in <module>
----> 1 plt.figure(); showSignatureVariances(anm_ens_20[:,0])

~/Documents/code/ProDy/prody/dynamics/signature.py in showSignatureVariances(mode_ensemble, **kwargs)
   1524     labels = ['mode %d'%(i+1) for i in indices][::-1]
   1525 
-> 1526     show = showSignatureDistribution(sig, label=labels, **kwargs)
   1527 
   1528     xlabel('Variance')

~/Documents/code/ProDy/prody/dynamics/signature.py in showSignatureDistribution(signature, **kwargs)
   1489     colors = []
   1490     for patch_i in patches:
-> 1491         colors.append(patch_i[0].get_facecolor())
   1492 
   1493     for i, patch_i in enumerate(patches):

TypeError: 'Polygon' object does not support indexing
```

**Comment by jamesmkrieger** [comment]
Now everything works fine.

File: prody/dynamics/signature.py [hunk 1]
```diff
@@ -1484,11 +1488,15 @@ def showSignatureDistribution(signature, **kwargs):
 
     colors = []
     for patch_i in patches:
-        colors.append(patch_i[0].get_facecolor())
+        if len(patches) == 1:
+            colors.append(patch_i.get_facecolor())
```
Comment (SHZ66) [inline prody/dynamics/signature.py:1492]: This block seems wrong. Please check

Comment (SHZ66) [reply prody/dynamics/signature.py:1492]: `if isinstance(patch_i, list):`

**Comment by jamesmkrieger** [comment]
This now works

**Pull request merged**